### ISSUE TITLE
Some clarifications about metrics reported in Table 2

### ISSUE THREAD

**Issue by morsingher** [WanquanF/NeuralPoints#12]
Hi, I am sorry to reiterate on this point but I am trying to reproduce your results and I have some questions about metrics reported in Table 2 of your paper. I attach the table here below:

![table_2_neuralpts](https://user-images.githubusercontent.com/36917382/167128758-33787505-5c46-4605-930e-797f42853784.png)

In section 4.1, you state: "We train and test our model on Sketchfab dataset collected by PUGeo-Net". Then, later in section 4.3: "For fairness, we share the same settings (batch size, iterations, learning rate, etc) among all methods". However, the corresponding table in the official PUGeo-Net paper (see Table 1 here: https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123640732.pdf) reports significantly different values, for all the methods included in both tables. 

![table_1_pugeonet](https://user-images.githubusercontent.com/36917382/167129301-f2ed9a06-31c7-4bb5-b398-4141b6192428.png)

In particular, the Chamfer Distance (CD) metric is about two orders of magnitude smaller. More specifically:
- PU-Net: 5.93 x 10e-5 vs 0.658 x 10e-2
- PUGeo-Net: 2.28 x 10e-5 vs 0.558 x 10e-2

To the best of my knowledge, your work and PUGeo-Net are the only **published** methods with metrics on the Sketchfab dataset, but neither of you have released the evaluation code, so it's hard to understand the reasons behind this difference. There is a recent ArXiV paper with a similar table on the same dataset (see Table 2 here: https://arxiv.org/pdf/2107.05893.pdf), which reports values very similar to PUGeo-Net and at least with the same order of magnitude. However, their code is not open source.

![table_2_puflow](https://user-images.githubusercontent.com/36917382/167130893-afb0c652-bea6-4465-9386-e372f23b4a95.png)

Therefore, I have three questions:
1) Can you please explain these differences and what they might be related to?
2) Can you please upload to this repository your evaluation code?
3) Can you please make your dataset accessible with another cloud service, other than Baidu?

Thank you in advance for your clarifications.

**Comment by WanquanF** [comment]
1. I think the reason should be the coordinate scale and whether to take the average for all points when computing the evaluation values.
2. Someone others also asked me to upload the evaluation script. I'll check it and upload it soon after finishing my current stuff.
3. I'll upload it to google drive.
